Extensions/Settings/Dnn.PersonaBar.Security/Security.Web/yarn.lock

```diff
@@ -1509,6 +1509,15 @@ create-hmac@^1.1.0, create-hmac@^1.1.2, create-hmac@^1.1.4:
     safe-buffer "^5.0.1"
     sha.js "^2.4.8"
 
+create-react-class@^15.6.3:
+  version "15.6.3"
+  resolved "https://www.myget.org/F/dnn-software-public/npm/create-react-class/-/create-react-class-15.6.3.tgz#2d73237fb3f970ae6ebe011a9e66f46dbca80036"
```

Comment:
Can you change to `https://registry.yarnpkg.com` like in the rest of the cases?

Reply:
@daguiler, fixed.